tests/python-tests/CWE-078/local/CommandInjectionLocal.expected

```diff
@@ -0,0 +1,18 @@
+edges
+| cmdi.py:4:5:4:28 | ControlFlowNode for input() | cmdi.py:7:17:7:17 | ControlFlowNode for i |
+| cmdi.py:4:5:4:28 | ControlFlowNode for input() | cmdi.py:9:17:9:30 | ControlFlowNode for Fstring |
+| cmdi.py:13:5:13:14 | ControlFlowNode for Attribute | cmdi.py:14:17:14:42 | ControlFlowNode for BinaryExpr |
+| cmdi.py:13:5:13:32 | ControlFlowNode for Attribute() | cmdi.py:14:17:14:42 | ControlFlowNode for BinaryExpr |
+nodes
+| cmdi.py:4:5:4:28 | ControlFlowNode for input() | semmle.label | ControlFlowNode for input() |
+| cmdi.py:7:17:7:17 | ControlFlowNode for i | semmle.label | ControlFlowNode for i |
+| cmdi.py:9:17:9:30 | ControlFlowNode for Fstring | semmle.label | ControlFlowNode for Fstring |
+| cmdi.py:13:5:13:14 | ControlFlowNode for Attribute | semmle.label | ControlFlowNode for Attribute |
+| cmdi.py:13:5:13:32 | ControlFlowNode for Attribute() | semmle.label | ControlFlowNode for Attribute() |
+| cmdi.py:14:17:14:42 | ControlFlowNode for BinaryExpr | semmle.label | ControlFlowNode for BinaryExpr |
+subpaths
+#select
+| cmdi.py:7:17:7:17 | ControlFlowNode for i | cmdi.py:4:5:4:28 | ControlFlowNode for input() | cmdi.py:7:17:7:17 | ControlFlowNode for i | This command depends on $@. | cmdi.py:4:5:4:28 | ControlFlowNode for input() | a user-provided value |
+| cmdi.py:9:17:9:30 | ControlFlowNode for Fstring | cmdi.py:4:5:4:28 | ControlFlowNode for input() | cmdi.py:9:17:9:30 | ControlFlowNode for Fstring | This command depends on $@. | cmdi.py:4:5:4:28 | ControlFlowNode for input() | a user-provided value |
+| cmdi.py:14:17:14:42 | ControlFlowNode for BinaryExpr | cmdi.py:13:5:13:14 | ControlFlowNode for Attribute | cmdi.py:14:17:14:42 | ControlFlowNode for BinaryExpr | This command depends on $@. | cmdi.py:13:5:13:14 | ControlFlowNode for Attribute | a user-provided value |
+| cmdi.py:14:17:14:42 | ControlFlowNode for BinaryExpr | cmdi.py:13:5:13:32 | ControlFlowNode for Attribute() | cmdi.py:14:17:14:42 | ControlFlowNode for BinaryExpr | This command depends on $@. | cmdi.py:13:5:13:32 | ControlFlowNode for Attribute() | a user-provided value |
```

Comment:
Strange that there are two lines here, I would have expected only one...

Reply:
This is gone now. Might have been an old commit